### PR TITLE
set crossScalaVersions in a more dbuild-friendly way

### PR DESCRIPTION
**Pull request by SethTisue**
needed to fix scala/community-builds#215

review by @sjrd

### CODE REVIEW

**Comment by SethTisue** [comment]
I'm afraid I don't completely understand what's going on here, but the new version seems just as reasonable to me as the old version, and the change does fix the community build issue.

**Comment by SethTisue** [comment]
I assume the Travis failure is because of some other recently merged PR?

**Comment by SethTisue** [comment]
investigating the Travis failure.

File: build.sbt [hunk 1]
```diff
@@ -20,7 +9,16 @@ lazy val `scala-parser-combinators` = crossProject.in(file(".")).
   settings(
     moduleName         := "scala-parser-combinators",
     version            := "1.1.0-SNAPSHOT",
-    scalaVersion       := crossScalaVersions.value.head
+    scalaVersion       := crossScalaVersions.value.head,
+    crossScalaVersions := {
```
Comment (sjrd) [inline build.sbt]: Hum, you can't do that. `crossScalaVersions` is *supposed* to be in `ThisBuild`. In subprojects, `crossScalaVersions` is just plain ignored, so effectively that's the same as not writing the setting at all. Hence the Travis failures.

**Comment by sjrd** [comment]
A possible alternate fix would be to change `scalaVersion := crossScalaVersions.value.head` into `scalaVersion := (crossScalaVersions in ThisBuild).value.head`. Or just plain hard-code `"2.11.7"`.

**Comment by SethTisue** [comment]
@sjrd how about 2692b8ab525d78b0f944f11fb4cbc2dd4b3a7cdf?

**Comment by sjrd** [comment]
LGTM

**Comment by SethTisue** [comment]
Travis is satisfied now and [dbuild seems happy with it as well](https://scala-ci.typesafe.com/job/scala-2.11.x-integrate-community-build/227/console)

**Pull request merged**